### DISCUSSION TITLE
Importing maven dependency; need repo listed?

### DISCUSSION THREAD

**Discussion by rfhoodrdm** [softcannery/camunda-formio-accelerator#109] [General]
I'm trying to set up a project that uses com.softcannery:camunda-formio-plugin:jar:0.0.16, but Maven complains it cannot locate this dependency. I imagine that this is due to me not having the right repositories set up in my settings.xml file for Maven. Is this the case? If not, can you point me in the right direction?

**Comment by softcannery-ltd** [comment]
Here is the link in Wiki with the instructions `How to add "camunda‐formio‐plugin" as a Maven dependency`:

- https://github.com/softcannery/camunda-formio-accelerator/wiki/How-to-add-%22camunda%E2%80%90formio%E2%80%90plugin%22-as-a-Maven-dependency
